package.json

```diff
@@ -117,6 +118,7 @@
     "@types/del": "3.0.1",
     "@types/fs-extra": "5.0.4",
     "@types/gulp": "4.0.5",
+    "@types/jest": "^24.0.25",
```

Comment:
I am very very wary about being in the business of managing yet another test frameworks.  If we're going to take ownership of these in this repo I think we need to convert these tests.

Reply:
(Resolved offline - the vscode-csharp repo now contains jest)